galley/pkg/runtime/monitoring/monitoring.go

```diff
@@ -154,6 +218,11 @@ func init() {
 		newView(processorEventsPerSnapshot, noKeys, view.Distribution(0, 1, 2, 4, 8, 16, 32, 64, 128, 256)),
 		newView(stateTypeInstancesTotal, collectionKeys, view.LastValue()),
 		newView(processorSnapshotLifetimesMs, noKeys, durationDistributionMs),
+		newView(stateTypeVirtualServicesTotal, nameKeys, view.Count()),
```

Comment:
are these truly `view.Count` or should they be `view.LastValue()`?  They should max out at `1`, right?

Reply:
Yeah it should max out at 1.. changed to view.LastValue